torch/ao/quantization/fx/convert.py

```diff
@@ -132,6 +132,7 @@ def _replace_observer_with_quantize_dequantize_node(
     if hasattr(activation_post_process, "compute_dtype"):
         compute_dtype = activation_post_process.compute_dtype  # type: ignore[attr-defined]
     quantize_op : Optional[Union[Callable, str]] = None
+    is_dynamic = False
     if dtype in [torch.quint8, torch.qint8, torch.qint32] and \
             not hasattr(activation_post_process, 'compute_dtype'):
```

Comment:
Not related to your changes but can we add some comments to explain what this case is? Is this static quantization?

Reply:
yes, this is static quantization